src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -867,6 +867,15 @@ public long ReadLastEtag()
             }
         }
 
+        public (long Etag, string ChangeVector) ReadLastEtagAndChangeVector()
+        {
+            using (DocumentsStorage.ContextPool.AllocateOperationContext(out DocumentsOperationContext documentsContext))
+            using (var tx = documentsContext.OpenReadTransaction())
+            {
+                return (DocumentsStorage.ReadLastEtag(tx.InnerTransaction), DocumentsStorage.GetDatabaseChangeVector(tx.InnerTransaction));
```

Comment:
Is there a match between the etag and the change vector?
_should_ there be?

Reply:
I think there should be a match.
But, does it really matter?